android/app/src/main/res/layout/activity_manual_control.xml

```diff
@@ -42,26 +42,38 @@
         app:layout_constraintTop_toTopOf="parent"></ImageView>
 
     <TextView
-        android:id="@+id/textView2"
+        android:id="@+id/speedHeader"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="24dp"
-        android:layout_marginLeft="24dp"
-        android:layout_marginTop="4dp"
-        android:text="Speed"
-        app:layout_constraintStart_toStartOf="@+id/joystickView2"
-        app:layout_constraintTop_toTopOf="@+id/joystickView2" />
+        android:layout_marginStart="4dp"
+        android:layout_marginLeft="4dp"
+        android:layout_marginBottom="8dp"
+        android:fontFamily="serif"
+        android:text="Speed:"
+        android:textColor="@color/teal_700"
+        android:textSize="20sp"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toTopOf="@+id/joystickView2"
+        app:layout_constraintStart_toStartOf="@+id/imageView"
+        app:layout_constraintTop_toTopOf="@+id/imageView"
+        app:layout_constraintVertical_bias="1.0" />
 
     <TextView
-        android:id="@+id/textView3"
+        android:id="@+id/angleHeader"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="4dp"
-        android:layout_marginEnd="24dp"
-        android:layout_marginRight="24dp"
-        android:text="Angle"
-        app:layout_constraintEnd_toEndOf="@+id/joystickView2"
-        app:layout_constraintTop_toTopOf="@+id/joystickView2" />
+        android:layout_marginEnd="4dp"
+        android:layout_marginRight="4dp"
+        android:layout_marginBottom="8dp"
+        android:fontFamily="serif"
+        android:text="Angle:"
+        android:textColor="@color/teal_700"
+        android:textSize="20sp"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toTopOf="@+id/joystickView2"
+        app:layout_constraintEnd_toEndOf="@+id/imageView"
+        app:layout_constraintTop_toTopOf="@+id/imageView"
+        app:layout_constraintVertical_bias="1.0" />
```

Comment:
Similar to the review above, I think changing lines 73 - 76 to the suggest code below may fix the text overlapping the camera view.
```suggestion
        app:layout_constraintBottom_toTopOf="@+id/joystickView2"
        app:layout_constraintEnd_toEndOf="@+id/imageView"
        app:layout_constraintTop_toBottomOf="@+id/imageView" />
```